ols/user_data_collection/data_collector.py

```diff
@@ -293,6 +293,15 @@ def gather_ols_user_data(data_path: str) -> None:
 
 
 if __name__ == "__main__":
+    # TODO: Temporary solution until OLS-500
+    try:
+        get_cloud_openshift_pull_secret()
+    except KeyError:
+        logger.error(
+            "Cluster pull secret does not contain the cloud.openshift.com auth"
+        )
+        sys.exit(1)
```

Comment:
pretty sure this is going to crashloop the container which is not what we want

it needs to run and just do nothing.